db/migrations/20221206081753_create_sizes_table.sql

```diff
@@ -0,0 +1,8 @@
+-- migrate:up
+CREATE TABLE sizes (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    size INT NOT NULL
```

Comment:
```suggestion
    size varchar(30) NOT NULL
```
sizes라는 테이블 특성상, 지금 기획으로는 신발 한정이라고 할 지라도 의류와 같은 다른 상품들의 경우도 확장하여 생각해본다면 integer보다 string type의 필드가 더 유용할 것 같습니다~